user-journey.adoc

```diff
@@ -1,62 +1,91 @@
-= User Journey for Istio/Jaeger
+= User Journey - Installing Istio + Fabric8 Launcher
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
+This document describes how to use the Istio installer to install the following components:
 
-== Flows description
+* Istio platform to connect, manage, and secure microservices.  This Istio installation includes the following components:
+** Elasticsearch
```

Comment:
Who added Elasticsearch? Why is this here?

Reply:
I believe that Elasticsearch is one of the Jaeger components.  At any rate, it's part of the installation.